### PR TITLE
Source File: Handle network errors

### PR DESCRIPTION
**Pull request by artem1205**
## What
Resolving https://github.com/airbytehq/oncall/issues/1743

## How

handle network errors

## Recommended reading order
2. `y.python`

## 🚨 User Impact 🚨

no breaking changes

### CODE REVIEW

File: airbyte-integrations/connectors/source-file/source_file/client.py [hunk 1]
```diff
@@ -396,6 +397,12 @@ def read(self, fields: Iterable = None) -> Iterable[dict]:
             except ConnectionResetError:
                 logger.info(f"Catched `connection reset error - 104`, stream: {self.stream_name} ({self.reader.full_url})")
                 raise ConnectionResetError
+            except ProtocolError as err:
```
Comment (bazarnov) [inline airbyte-integrations/connectors/source-file/source_file/client.py:400]: Do we have the unit_test for this  case?

Comment (artem1205) [reply airbyte-integrations/connectors/source-file/source_file/client.py:400]: unit test added

**Review by bazarnov — APPROVED** [review]
Changelog, version bump, and new checklist are waiting for you.

**Review by lazebnyi — APPROVED** [review]
With changes asked by @bazarnov looks good to me.

**Pull request merged**